### PR TITLE
Add changelog and docs for v1.5 release

### PR DESCRIPTION
**Pull request by nrb**
Signed-off-by: Nolan Brubaker <brubakern@vmware.com>

### CODE REVIEW

File: site/content/docs/v1.5/_index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ If you encounter issues, review the [troubleshooting docs][30], [file an issue][
 
 ## Contributing
 
-If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.5.0-rc.1/start-contributing/) documentation for guidance on how to setup Velero for development.
+If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.5.1/start-contributing/) documentation for guidance on how to setup Velero for development.
```
Comment (carlisia) [inline site/content/docs/v1.5/_index.md:36]: s/v1.5.1/v1.5
```suggestion
If you are ready to jump in and test, add code, or help with documentation, follow the instructions on our [Start contributing](https://velero.io/docs/v1.5/start-contributing/) documentation for guidance on how to setup Velero for development.
```

File: site/content/docs/v1.5/contributions/oracle-config.md [hunk 2]
```diff
@@ -244,5 +244,5 @@ After creating the Velero server in your cluster, try this example:
 
 ## Additional Reading
 
-* [Official Velero Documentation](https://velero.io/docs/v1.5.0-rc.1/)
+* [Official Velero Documentation](https://velero.io/docs/v1.5.1/)
```
Comment (carlisia) [inline site/content/docs/v1.5/contributions/oracle-config.md:247]:
```suggestion
* [Official Velero Documentation](https://velero.io/docs/v1.5/)
```

File: site/content/docs/v1.5/release-instructions.md [hunk 3]
```diff
@@ -6,57 +6,57 @@ toc: "true"
...
-6. Clean up when there is an existing set of pre-release versioned docs for the version you are releasing
+1. Clean up when there is an existing set of pre-release versioned docs for the version you are releasing
 	- Example: `site/content/docs/v1.5.0-beta.1` exists, and you're releasing `v1.5.0-rc.1` or `v1.5`
     - Remove the directory containing the pre-release docs, i.e. `site/content/docs/<pre-release-version>`.
     - Delete the pre-release docs table of contents file, i.e. `site/data/<pre-release-version>-toc.yml`.
     - Remove the pre-release docs table of contents mapping entry from `site/data/toc-mapping.yml`.
     - Remove all references to the pre-release docs from `site/config.yml`.
-7. Create/update the "Upgrade to $major.minor" page.
-8. Review and submit PR
+1. Create/update the "Upgrade to $major.minor" page (Example: https://velero.io/docs/v1.5.1/upgrade-to-1.5/). This needs to be done in both the versioned and the `main` folders.
```
Comment (carlisia) [inline site/content/docs/v1.5/release-instructions.md:49]:
```suggestion
1. Create/update the "Upgrade to $major.minor" page (Example: https://velero.io/docs/v1.5/upgrade-to-1.5/). This needs to be done in both the versioned and the `main` folders.
```

**Review by carlisia — CHANGES_REQUESTED** [review]

**Review by carlisia — APPROVED** [review]
👍

**Review by zubron — APPROVED** [review]

**Pull request merged**